src/test/java/salesforce/steps/GeneralEndPointSteps.java

```diff
@@ -267,4 +258,12 @@ public void iSetTheEndpointAndSendTheRequestWithBody(final String endpoint) {
         apiRequest.setEndpoint(endpoint);
```

Comment:
This @When is the equals to the line 262

Reply:
Ok Gus, is done